applications/views.py

```diff
@@ -192,7 +190,9 @@ def page(request, pk_hash, key):
     # check the user can access this application
     validate_application_access(request.user, application)
 
-    if application.approved_at and not has_role(request.user, StaffAreaAdministrator):
+    if application.approved_at and not has_permission(
+        request.user, permissions.staff_area_access
```

Comment:
thought: I've tried to keep the diffs minimal, and easy to understand, because there are a large number of changes across multiple files.

So:

* sometimes we're using `permissions` as an import, because we already had that imported, and then accessing `staff_area_access` on that
* other times, we're importing `staff_area_access` directly and using that, which is an exchange for `StaffAreaAdministrator`

If it's thought preferable to make all the imports using the permissions consistent, perhaps to explicitly mark them as `permissions.staff_area_access`, that's also fine.

Though, it may be that the permissions get refactored into a class in #5459 subsequently, so I'm not sure it's worth worrying about too much now.